soh/src/overlays/actors/ovl_En_Ossan/z_en_ossan.h

```diff
@@ -14,23 +14,6 @@ typedef void (*EnOssanInitFunc)(struct EnOssan*, GlobalContext*);
 typedef s16 (*EnOssanGetGirlAParamsFunc)(s16);
 typedef void (*EnOssanStateFunc)(struct EnOssan*, GlobalContext*, Player*);
 
-typedef struct {
-    /* 0x00 */ u32 stickColorR;
-    /* 0x04 */ u32 stickColorG;
-    /* 0x08 */ u32 stickColorB;
-    /* 0x0C */ u32 stickColorA;
-    /* 0x10 */ f32 stickTexX;
-    /* 0x14 */ f32 stickTexY;
-    /* 0x18 */ u32 arrowColorR;
-    /* 0x1C */ u32 arrowColorG;
-    /* 0x20 */ u32 arrowColorB;
-    /* 0x24 */ u32 arrowColorA;
-    /* 0x28 */ f32 arrowTexX;
-    /* 0x2C */ f32 arrowTexY;
-    /* 0x30 */ f32 z;
-    /* 0x34 */ s32 isEnabled;
-} StickDirectionPrompt; // size = 0x38
```

Comment:
Did you mean to remove this?

Reply:
Yes it got moved to z64.h and it didn't like them being in both places. En_Ossan should still be able to access that struct from there, but now that you mention it I haven't explicitly double checked that the shops still work, should do that.